src/bson.rs

```diff
@@ -211,9 +209,9 @@ impl From<bool> for Bson {
     }
 }
 
-impl From<(String, String)> for Bson {
```

Comment:
I'm not thrilled with the previous implementation of converting from `(String, String)` for regex, `(String, Document)` for CodeWithScope, etc., as there might eventually be a new BSON type that's isomorphic to an existing one, so I replaced each of these with the conversions from the new structs.